cmd/flags/create.go

```diff
@@ -74,7 +75,15 @@ func runCreate(client flags.Client) func(*cobra.Command, []string) error {
 			return err
 		}
 
-		fmt.Fprintf(cmd.OutOrStdout(), string(response)+"\n")
+		output, err := output.CmdOutput(
```

Comment:
@k3llymariee At some point for the dynamic endpoint function, we'll need a field that defines which function to call for each action.